### PR TITLE
[Release] Core_Java

### PR DESCRIPTION
**Pull request by Playjasb2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,62 @@
...
...puter Science Specialist and Math Minor), and I intend to venture into software development and data engineering. With my strong background in mathematics, it became natural that I am centered around problem-solving, and being able to construct elegant solutions that are extendable, scalable, and maintainable. In terms of academics, I have studied operating systems, algorithms, computability, web development, parallel programming, scalable computing, machine learning, deep learning, and much more. I have experience in projects involving compilers, web programming, scripting, and data visualization. As for work experience, I had worked as a software developer in helping a firm migrate its frontend services, and I currently work at Jarvis as a data engineer, where I work on projects that involve constructing some pipelines to deliver data in a format that would allow for further analysis. Altogether, they make up all the skills and experience that I want to carry forward towards new opportunities. New opportunities that will make use of my abilities and allow me to further learn new technologies. I am passionate about this journey that I'm on, and I can't wait to see what's up ahead.
```
Comment (jarviscanada) [inline README.md:3]: Check your spelling and grammar through https://www.grammarly.com/

File: README.md [hunk 2]
```diff
@@ -1,8 +1,62 @@
...
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar/tree/master/linux_sql)]: Implemented a monitoring agent that can track the hardware specifications and usage throughout time of host machines and store their data in a database. The monitoring agent is programmed using Bash, and Postgres is used as a database, which would run in a Docker container. Testing was all done on a CentOS 7 virtual machine running on the Google Cloud Platform. Every script was tested manually to ensure that they all adhere to their specifications.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar/tree/master/core_java)]:
+      
+  - Twitter App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+  - JDBC App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+  - Grep App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
```
Comment (jarviscanada) [inline README.md:24]: don't forget to update those when you finished the project

File: core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+        JavaGrepImp javaGrepImp = new JavaGrepImp();
+        javaGrepImp.setRegex(args[0]);
+        javaGrepImp.setRootPath(args[1]);
+        javaGrepImp.setOutFile(args[2]);
+
+        try {
+            javaGrepImp.process();
+        } catch(Exception e) {
+            javaGrepImp.logger.error(e.getMessage(), e);
```
Comment (jarviscanada) [inline core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java]: javaGrepImp.logger.error(~e.getMessage()~ "meaningful message", e);

File: core_java/grep/src/main/java/ca/jrvs/apps/practice/HelloWorld.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package ca.jrvs.apps.practice;
+
+class HelloWorld {
+
+    // Your program begins with a call to main().
```
Comment (jarviscanada) [inline core_java/grep/src/main/java/ca/jrvs/apps/practice/HelloWorld.java]: remove this file

**Review by jarviscanada — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (Playjasb2) [reply README.md:3]: Done.

File: README.md (hunk 2 above)
Comment (Playjasb2) [reply README.md:24]: Alright, I updated the descriptions for Grep and JDBC.

File: core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java (hunk 3 above)
Comment (Playjasb2) [reply core_java/grep/src/main/java/ca/jrvs/apps/grep/JavaGrepImp.java]: Fixed it.

File: core_java/grep/src/main/java/ca/jrvs/apps/practice/HelloWorld.java (hunk 4 above)
Comment (Playjasb2) [reply core_java/grep/src/main/java/ca/jrvs/apps/practice/HelloWorld.java]: Done.

**Pull request merged**